services/ui-src/src/components/fields/ChoiceField.tsx

```diff
@@ -1,51 +1,86 @@
 import { useFormContext } from "react-hook-form";
 // components
 import { Choice as CmsdsChoice } from "@cmsgov/design-system";
-import { Box } from "@chakra-ui/react";
+import { Box, Text } from "@chakra-ui/react";
 // utils
 import { makeMediaQueryClasses } from "utils";
 import { InputChangeEvent, AnyObject } from "types";
+import { useEffect, useState } from "react";
 
-export const ChoiceField = ({
-  name,
-  type,
-  value,
-  label,
-  sxOverride,
-  ...props
-}: Props) => {
+export const ChoiceField = ({ name, label, sxOverride, ...props }: Props) => {
   const mqClasses = makeMediaQueryClasses();
+  const singleBoxClass = "single-box";
+
+  const [displayValue, setDisplayValue] = useState<boolean>(false);
 
   // get the form context
   const form = useFormContext();
+  form.register(name);
+
+  // set initial display value to form state field value or hydration value
+  const hydrationValue = props?.hydrate;
+  useEffect(() => {
+    // if form state has value for field, set as display value
+    const fieldValue = form.getValues(name);
+    if (fieldValue) {
+      setDisplayValue(fieldValue);
+    }
+    // else if hydration value exists, set as display value
+    else if (hydrationValue) {
+      setDisplayValue(hydrationValue);
+      form.setValue(name, hydrationValue, { shouldValidate: true });
+    }
+  }, [hydrationValue]); // only runs on hydrationValue fetch/update
 
   // update form data
   const onChangeHandler = async (event: InputChangeEvent) => {
-    const { name: choiceName, value: choiceValue } = event.target;
-    form.setValue(choiceName, choiceValue, { shouldValidate: true });
+    const { name: choiceName } = event.target;
+    form.setValue(choiceName, !displayValue, { shouldValidate: true });
+    setDisplayValue(!displayValue);
   };
 
+  const componentProps = { ...props, hydrate: "" };
   return (
-    <Box sx={{ ...sx, ...sxOverride }} className={mqClasses}>
+    <Box
+      sx={{ ...sx, ...sxOverride }}
+      className={`${mqClasses} ${singleBoxClass}`}
+    >
+      <Text sx={sx.label} id="label">
+        {label}
+      </Text>
       <CmsdsChoice
         name={name}
-        type={type}
-        value={value}
-        label={label}
+        label={<></>}
```

Comment:
^^ This is necessary because CMSDS `Choice` requires either a valid string or React child(ren) to be passed to `FormLabel`. Passing just an empty string would be great, but you can't do that for a `Choice` component, and not having a `label` at all makes it mad too. There's also accessibility to be considered, so we can't just render the `label` once in the `Text` above and once in `label` and hide it, because screen readers would read it twice, so this is where we ended up.